### ISSUE TITLE
Feature Request: Mount devices written as a TAR

### ISSUE THREAD

**Issue by inthebrilliantblue** [mxmlnkn/ratarmount#118]
I currently have about 20-ish hard drives that were written to using tar -M, meaning all these drives are technically one big tar archive. I can pull file information one by one through the usual tar -tvf command, but would like to use ratarmount to see if it would be faster to use these drives.

It appears ratarmount was written to work with files only, but it would be nice to be able to use any tape drive or hard drive archive as well. Is there any reason why working with a device would not work, even if its read only?

**Comment by mxmlnkn** [comment]
No, there isn't any reason per se except that it isn't supported yet and that I never used multi-volume TAR archives. What exactly does the `tar -tvf` command look like? You speak of devices, so does the command look something like `tar -tvf -M /dev/sdc /dev/sdd ...`?

**Comment by inthebrilliantblue** [comment]
> No, there isn't any reason per se except that it isn't supported yet and that I never used multi-volume TAR archives. What exactly does the `tar -tvf` command look like? You speak of devices, so does the command look something like `tar -tvf -M /dev/sdc /dev/sdd ...`?

So the tar -tvf command will read the tar device, but then exit on an error if and when a file is split between two tar volumes with M not specified. It will show the incomplete filename that is split in the error. tar -tvMf /dev/sdc will output the same list but then ask for the next volume via the terminal.
